File: Data Structures/Trees/Postorder Traversal.cpp
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
	*/
void postOrder(node* root) {
 if(root != NULL)
    {

        postOrder(root->left);
        postOrder(root->right);
        cout << root->data<<" ";

 }}

File: Data Structures/Trees/Postorder Traversal.java
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
	*/
static void postOrder(Node root) {
 if(root != null)
    {postOrder(root.left);
        postOrder(root.right);
        System.out.print(root.data+"\t");
     

    }
}
